components/org.wso2.carbon.sts/src/main/java/org/wso2/carbon/sts/STSDeploymentInterceptor.java

```diff
@@ -208,7 +210,11 @@ public static void updateSTSService(AxisConfiguration config) throws Exception {
             stsSamlConfig.setAddRequestedUnattachedRef(true);
             stsSamlConfig.setKeyComputation(2);
             stsSamlConfig.setProofKeyType(TokenIssuerUtil.BINARY_SECRET);
-            stsSamlConfig.setCallbackHandlerName(AttributeCallbackHandler.class.getName());
+            if (StringUtils.isNotBlank(callBackHandlerName)) {
+                Class clazz = Thread.currentThread().getContextClassLoader().loadClass(callBackHandlerName);
+                callBackHandlerClassName = clazz.newInstance().getClass().getName();
+                stsSamlConfig.setCallbackHandlerName(callBackHandlerClassName);
+            }
 
```

Comment:
There's no need to load the class dynamically here. Setting the full qualified class name read from the config would be enough. We can remove the dynamic import also in this case.